public/styles/styles.css

```diff
@@ -17,5 +17,23 @@ h1 {
 .vehicle-image {
 	height: 210px;
 	width: 320px;
+  border-radius: 20px;
 }
 
+.whole-vehicle-card {
+  margin: 10px;
+  border-radius: 20px;
+}
+
+.categories-nav-buttons {
+  align-content: center;
+  padding: 5px 20px;
+  margin-right: 50px;
+}
+
+.edit-btn, .delete-btn, .save-btn  {
+  margin: 10px;
+  width: 100px;
+  border-radius: 20px;
+  align-self: center;
+}
```

Comment:
missing newline at end of file

Reply:
@mnfmnfm I am unsure what you mean by a "missing newline at the end of the file"